### DISCUSSION TITLE
Questions regarding implementation of CAGEE

### DISCUSSION THREAD

**Discussion by agneeshbarua** [hahnlab/CAGEE#20] [Q&A] [answered]
Greetings,
 
 I have been using CAGEE on my data and had some questions about its implementation.

I have a set of genes, gene set-1. I hypothesise that gene expression of gene set-1 evolved differently in specific lineages of interest. So I fit a two-rate model, with my lineages as test branches, and compared it with the single-rate (null) model. I found that the two-rate model fits better. This tells me that gene set-1 experienced divergent evolution in my test lineages. 

I had the following questions: 

Is there a way to know whether gene expression evolution of gene set-1 in my test branches is because the genes themselves have some functional relevance in the test branches, or is it because of the tree's topology and distribution of the test lineages? For instance, I would like to know whether it makes sense to use genes, not in gene set-1 and see if the two-rate model still fits better. If the two-rate model still fits better using the alternate gene set, does it mean my choice of genes is irrelevant? 

When I ran CAGEE on my data, many genes significantly decreased their expression through evolution. Could this be interpreted as purifying selection refining the gene expression of certain genes, thereby making changes in their expression less tolerated throughout evolution?  

Any suggestions and tips would be really appreciated.

**Comment by hahnlab-user** [comment, marked as answer]
Hi Agneesh,

Thanks for using our tool! A few answers:

1) I would not say that finding a better fit for a two-rate model indicates divergent evolution. Simply that a more complex model fits better.
2) There are a couple of ways to test whether your genes and branches are "special" in some way. One simple way would be to pick the sister lineages to your test branches, and assign them all their own rate. Then, compare a two-rate model with these sisters and the background branches to a one-rate model. If this is also significant, it might simply mean that there is a lot of rate heterogeneity in nature. You could of course also choose a second gene set, to see again if there's rate heterogeneity among lots of different types of genes.
3) I don't think decreases in expression indicate purifying selection.

Hope that helps,
Matt

**Comment by agneeshbarua** [comment]
Thank you for the reply. Things are clear now!
